app/repository/document.py

```diff
@@ -404,6 +404,7 @@ def create(db: Session, document: DocumentCreateDTO) -> str:
 
         org_name = cast(str, org.name)
 
+        # TODO: Fix this call - the transaction is changed here
```

Comment:
Do we still need this? If so, can we link a ticket please?

Reply:
yeah that would be good ...

Reply:
... okay I've removed the comment. But if we do want to start using the quick tests again - then we need to ensure that we pass the current transaction into the `db-client` ... also creating a ticket...

Reply:
https://linear.app/climate-policy-radar/issue/PDCT-1197/possible-rollback-issue